accessibility/index.html

```diff
@@ -0,0 +1,758 @@
+                </section
+				
+				<!-- Exercise 01 START -->
+
+				<section>
+					<h3>Exercise One</h3>
+          			<p>Use VoiceOver to read content that uses <br/><strong>Non-Semantic</strong> HTML elements</p>
+		  			<small><a href="https://inclusiveThinking.jsbin.com/mudiwu/1/edit?html,css,output" target="_blank">Open Exercise</a></small>
+                    <aside class="notes">
+						Open Code in JS Bin's Live Preview before using screen reader
+					</aside>
+				</section>
+
+				<!-- Exercise 01 END -->
+
+				<!-- Exercise 02 START -->
+
+				<section>
+					<h3>Exercise 2</h3>
+          			<p>Modify the HTML using <strong>Semantic</strong> HTML elements <br/>and read the page again</p>
+		  			<small><a href="https://inclusiveThinking.jsbin.com/dihebu/3/edit?html,css,output" target="_blank">Modify HTML from Exercise One</a></small>
+                     <aside class="notes">
+						Have students modify the HTML using semantic elements then open Code in JS Bin's Live Preview before using screen reader. Students will now be hear a difference in the way the HTML is identified by a screen reader
+					</aside>
+				</section>
+				</section>
+
+				<!-- Exercise 02 END -->
+                    
+                <section>
+                    <h2>Using <span class="gdi">Browser Plugins</span> for <span class="gdi">Accessibility Testing</span></h2>
+                    <p>The <span class="gdi">WAVE</span> browser extension from WebAIM allows you to evaluate web content for accessibility issues directly within the browser. It is available for  Chrome and FireFox.</p>
+		  			<small><a href="http://wave.webaim.org/extension/" target="_blank">WAVE Web Accessibilty Evaluation Tool</a><br>(The way the text above is linked is intentional and is an example of link context from the WCAG which we will discuss later)</small>
+                    <p>Once installed a "W" icon will appear in the browser toolbar. Press the icon to run an audit of the page. Errors and warnings will appear within the page. Click the error or warning to see more information in the left of the browser.</p>
+					</aside>
+				</section>
+            
+                <section>
+                    <h2>Using <span class="gdi">Browser Plugins</span> for <span class="gdi">Accessibility Testing</span></h2>
+                    <p><span class="gdi">tota11y</span> is an accessibility visualization toolkit from Khan Academy. It can be added to Chrome from the Chrome Web Store</p>
+		  			<small><a href="http://khan.github.io/tota11y/" target="_blank">tota11y Accessibility Visualization Toolkit</a></small>
+				</section>
+
+				<!-- Exercise 03 START -->
+			
+				<section>
+					<h3>Exercise 3</h3>
+          			<p>Use an accessibility testing plugin to identify and correct the errors from the automated test.</p>
```

Comment:
Yes, I clarified and added a link to the associated JSbin page

On Mon, Oct 26, 2015 at 9:45 PM, Brenda Jin <notifications@github.com>
wrote:

> In accessibility/index.html
> <https://github.com/pamelafox/teaching-materials/pull/169#discussion_r43084043>
> :
>
> > +		  			<small><a href="http://wave.webaim.org/extension/" target="_blank">WAVE Web Accessibilty Evaluation Tool</a><br>(The way the text above is linked is intentional and is an example of link context from the WCAG which we will discuss later)</small>
> > +                    <p>Once installed a "W" icon will appear in the browser toolbar. Press the icon to run an audit of the page. Errors and warnings will appear within the page. Click the error or warning to see more information in the left of the browser.</p>
> > +					</aside>
> > +				</section>
> > +
> > +                <section>
> > +                    <h2>Using <span class="gdi">Browser Plugins</span> for <span class="gdi">Accessibility Testing</span></h2>
> > +                    <p><span class="gdi">tota11y</span> is an accessibility visualization toolkit from Khan Academy. It can be added to Chrome from the Chrome Web Store</p>
> > +		  			<small><a href="http://khan.github.io/tota11y/" target="_blank">tota11y Accessibility Visualization Toolkit</a></small>
> > +				</section>
> > +
> > +				<!-- Exercise 03 START -->
> > +			
> > +				<section>
> > +					<h3>Exercise 3</h3>
> > +          			<p>Use an accessibility testing plugin to identify and correct the errors from the automated test.</p>
>
> What should they be testing? The previous jsbin page?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pamelafox/teaching-materials/pull/169/files#r43084043>
> .
>